test/irmin-pack/layered.ml

```diff
@@ -292,7 +292,11 @@ module Test = struct
     Store.sync ro_ctxt.index.repo;
     Log.debug (fun l -> l "Freeze removes block1b but keeps block1, block1a");
     let* ctxt = freeze ctxt block1a in
+    if not (Store.async_freeze ro_ctxt.index.repo) then
```

Comment:
What if the freeze finishes before this statement is reached?

Reply:
Thanks, good point!